parts/nav-bfc-submenu.php

```diff
@@ -16,6 +15,23 @@
 
 		<?php endwhile; ?>
 
+		<li id="more" class="<?php bp_nouveau_nav_classes(); ?>">
+		<span class="menu-more" data-toggle="more-dropdown">More ...</span>
+			<div class="dropdown-pane" id="more-dropdown" data-dropdown data-hover="true" data-hover-pane="true" data-auto-focus="true">
+				<ul>
+					<?php
+					while ( bp_nouveau_nav_items() ) :
+						bp_nouveau_nav_item();
+					?>
+						<li id="<?php bp_nouveau_nav_id(); ?>" class="<?php bp_nouveau_nav_classes(); ?>">
```

Comment:
Seems like you can just add some additional classes here to help with the styling.

You could also add a counting variable to your while so you can tell which item you're on and do special processing that way as needed.

Reply:
Thanks for the pointers. I'll see what I can do for the next PR.